Feat/migrate search delete emails o365 to msg (#44461)
demisto/content: Packs/CommonPlaybooks/ReleaseNotes/2_8_0.md

#### Playbooks

##### Search And Delete Emails - Generic v2

> - Updated the Search And Delete Emails - Generic v2 playbook to support the new Search And Delete Emails - Microsoft Graph Security playbook.

Reviewer: ```suggestion
- Updated the **Search And Delete Emails - Generic v2** playbook to support the new **Search And Delete Emails - Microsoft Graph Security** playbook.
```

> 

Reviewer: ```suggestion
```

> **Breaking Change:** The **Search And Delete Emails - Generic v2** playbook no longer uses the deprecated O365 Security & Compliance (Search And Delete Emails - Office 365) sub-playbook. It has been replaced with the new **Search And Delete Emails - Microsoft Graph Security** sub-playbook, which uses the Microsoft Graph eDiscovery API. Installations relying on the O365 S&C integration for email deletion must migrate to the Microsoft Graph Security integration.

Reviewer: ```suggestion
- **Breaking Change:** The **Search And Delete Emails - Generic v2** playbook no longer uses the deprecated **O365 - Security And Compliance - Content Search v2** integration commands (via the **Search And Delete Emails - Office 365** sub-playbook). It now uses the **Search And Delete Emails - Microsoft Graph Security** sub-playbook, which utilizes the Microsoft Graph eDiscovery API. Cortex installations that use the **O365 - Security And Compliance - Content Search v2** integration for email deletion must migrate to the **Microsoft Graph Security** integration.
```
